grpcurl.go

```diff
@@ -345,9 +344,8 @@ func InvokeRpc(ctx context.Context, source DescriptorSource, cc *grpc.ClientConn
 		return invokeClientStream(ctx, stub, mtd, handler, requestData, req)
 	} else if mtd.IsServerStreaming() {
 		return invokeServerStream(ctx, stub, mtd, handler, requestData, req)
-	} else {
-		return invokeUnary(ctx, stub, mtd, handler, requestData, req)
 	}
+	return invokeUnary(ctx, stub, mtd, handler, requestData, req)
```

Comment:
This being at a different indentation level from the others looks funny to me. I bet we can change that and still keep the linter happy by changing this to a `switch` where this is the default case.